### PR TITLE
Fixes #18534: Skip non-standard moint FS for free-space check

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/18534

### CODE REVIEW

**Comment by fanf** [comment]
No, you need to first filter to keep only lines starting with "/" and then keep the original logic of looking for "/var" or use "/"

**Review by fanf — CHANGES_REQUESTED** [review]
No, you need to first filter to keep only lines starting with "/" and then keep the original logic of looking for "/var" or use "/"

**Comment by ElaadF** [comment]
Commit modified

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/healthcheck/HealthcheckService.scala [hunk 1]
```diff
@@ -96,7 +96,7 @@ final object CheckFreeSpace extends Check {
 
   def run: IOResult[HealthcheckResult] = {
     val file          = root / "proc" / "mounts"
-    val mountsContent = file.lines.map(x => x.split(" ")(1)).toList
+    val mountsContent = file.lines.filter(_(0) == '/').map(x => x.split(" ")(1)).toList
```
Comment (fanf) [inline webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/healthcheck/HealthcheckService.scala:99]: can you had a comment to explain? With a link to the bug for future reference ?

**Review by fanf — CHANGES_REQUESTED** [review]

**Comment by ElaadF** [comment]
Commit modified

**Review by fanf — APPROVED** [review]

**Pull request merged**